docs/copilot/workspace-context.md

```diff
@@ -0,0 +1,72 @@
+
+First, `@workspace` uses GitHub Copilot to disambiguate the user's prompt and to get input for the different search strategies that will produce the right amount of context.
+
+Next, it performs a combination of search strategies, such as [GitHub's code search index](https://github.blog/2023-02-06-the-technology-behind-githubs-new-code-search), a lexical text searches over the local index to find local, uncommitted changes, and VS Code's language intelligence to add details like function signatures, parameters, and more.
+
+Finally, all of these pieces of context are ranked, sliced, and summarized by `@workspace`, and then sent off to GitHub Copilot to answer the user's request.
+
+After the Copilot response is received, `@workspace` processes the response and adds references to files, file ranges, and symbols. This enables users to link directly from the chat response to the corresponding information in their codebase.
+
+## Tips for using @workspace
+
+- Be specific in your questions and use terms and concepts that are potentially present in your codebase
+- Use chat variables to expand the explicit context on top of what is automatically included from the codebase, such as `#editor`, `#selection`, or `#file`
+- Don't expect statistical analysis, such as "how many times is this function called?" or "how many times is this variable used?"
+
+## Context for @workspace slash commands
+
+If you're using slash commands, then they define their own context that is optimized for their specific task:
+
+| Command    | Context |
+| ---------- | ------- |
+| `/explain` | <ul><li>Starts with the text selection in the active editor (#selection). To optimize the Copilot chat responses, make sure to expand the text selection to include any relevant information to help Copilot provide a useful response.</li></ul><ul><li>Looks up the implementations of referenced symbols such as functions and classes, leading to more accurate and useful explanations.</li></ul> |
+| `/test`    | <ul><li>Current text selection in the active editor. If no text is selected, use the contents of the currently active file.</li></ul><ul><li>Related existing test files, to understand existing tests and best practices.</li></ul> |
+| `/fix`     | <ul><li>Current text selection in the active editor. If no text is selected, use the currently visible text in the editor.</li></ul><ul><li>Errors and referenced symbols to understand what needs to be fixed and how.</li></ul> |
+
+You can explicitly expand the context by using chat variables, such as `#editor`, `#selection`, or `#file` in your chat prompt. For example, to get an explanation of the currently visible code in the editor, use this chat prompt: `@workspace /explain #editor`.
```

Comment:
I think this explanation of `#variables` is what I was looking for earlier in the doc!